code/game/gamemodes/cult/ritual.dm

```diff
@@ -273,6 +271,8 @@
 				N.icon_state = "shield-cult"
 				N.health = 60
 				shields |= N
+		else
+			return//don't do shit

```

Comment:
As a note to other nerds: This should fix "Finale runes aren't getting announced"